src/test/java/org/xstefank/whitelist/AddUserCommandTest.java

```diff
@@ -0,0 +1,79 @@
+package org.xstefank.whitelist;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import java.io.File;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.powermock.api.mockito.PowerMockito;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
+import org.xstefank.ci.CILoader;
+import org.xstefank.model.PersistentList;
+import org.xstefank.model.Utils;
+import org.xstefank.TestUtils;
+import org.xstefank.api.GitHubAPI;
+
+import static org.powermock.api.support.membermodification.MemberMatcher.method;
+
+@RunWith(PowerMockRunner.class)
+@PrepareForTest({CILoader.class, GitHubAPI.class})
+public class AddUserCommandTest {
+
+    private static final File userListFile = new File(TestUtils.TARGET_DIR, Utils.USERLIST_FILE_NAME);
+    private static final TestCI testCI = new TestCI();
+
+    private final PersistentList userList = new PersistentList(userListFile);
+    private final PersistentList adminList = new PersistentList(TestUtils.RESOURCES_PATH, Utils.ADMINLIST_FILE_NAME);
+    private AddUserCommand addUserCommand;
+
+    @BeforeClass
+    public static void beforeClass() {
+        TestUtils.deleteFileIfExists(userListFile);
+        PowerMockito.stub(method(CILoader.class, "getCI", String.class))
+                .toReturn(testCI);
+        WhitelistProcessing.loadCIs(TestUtils.FORMAT_CONFIG);
+    }
+
+    @Before
+    public void before() {
+        PowerMockito.stub(method(GitHubAPI.class, "getJsonWithPullRequest", JsonNode.class))
+                .toReturn(TestUtils.TEST_PAYLOAD);
+        addUserCommand = new AddUserCommand();
+    }
+
+    @Test
+    public void testAddUserToList() {
+       addUserCommand.process(TestUtils.ISSUE_PAYLOAD, adminList, userList);
+        Assert.assertTrue(TestUtils.fileContainsLine(userListFile,"prUser"));
+    }
+
+    @Test
+    public void testTriggerBuildMethodCall() {
+        String prAuthor = TestUtils.ISSUE_PAYLOAD.get(Utils.ISSUE).get(Utils.USER).get(Utils.LOGIN).asText();
+        userList.addUser(prAuthor);
+        addUserCommand.process(TestUtils.ISSUE_PAYLOAD, adminList, userList);
+        Assert.assertTrue(testCI.isTriggered());
+    }
+
+    @Test
+    public void testCommandRegexValue() {
+        String regex = TestUtils.FORMAT_CONFIG.getFormat().getCommands().get(AddUserCommand.NAME);
+        addUserCommand.setCommandRegex(regex);
+        Assert.assertEquals(regex, addUserCommand.getCommandRegex());
```

Comment:
this test tests only that if you set value with setter you will get the same back with getter? There is nothing wrong with the test, it's just a little unnecessary